tests/ui/abi/cannot-be-called.rs

```diff
@@ -19,15 +19,11 @@
 #![feature(
     no_core,
     lang_items,
-    abi_ptx,
+    abi_custom,
```

Comment:
This ABI apparently isn't in this test either, actually.

Reply:
Right, I removed that because it's already tested elsewhere and dealing with the assembly of a naked function on top of the revisions is kind of annoying.